tests/unit/proxy/test_elasticsearch.py

```diff
@@ -93,7 +95,8 @@ def setUp(self) -> None:
                                              column_names=['test_col1', 'test_col2'],
                                              tags=self.mock_empty_tag,
                                              badges=self.mock_empty_badge,
-                                             last_updated_timestamp=1527283287)
+                                             last_updated_timestamp=1527283287,
+                                             programmatic_descriptions=[])
```

Comment:
Could you add a test case where programmatic_descriptions is `None` or not being assigned to see if it is backward compatible?

Reply:
ok will do, good call out.

Reply:
Ok added a case where it is None as well just in case. Also confirmed in the table.py programmatic_descriptions defaults to [] not None.

Reply:
hmm ok, no that won't work because the type of programmatic_description is list[string]. It has a default value of [] if not set. Also, this test uses mock search results.